main.ts

```diff
@@ -302,14 +302,36 @@ function openThisDamnFile(pathToVhaFile: string) {
         'finalObjectReturning',
         lastSavedFinalObject,
         pathToVhaFile,
-        globals.selectedOutputFolder + path.sep,   // app needs the trailing slash (at least for now)
+        getHtmlPath(globals.selectedOutputFolder),
         changedRootFolder,
         rootFolderLive
       );
     }
   });
 }
 
+/**
+ * Return an HTML string for a path to the `vha` file
+ * e.g. `C:\Some folder` becomes `C:/Some%20folder`
+ * @param anyOsPath
+ */
+function getHtmlPath(anyOsPath: string): string {
+      // Windows was misbehaving
+      // so we normalize the path (just in case) and replace all `\` with `/` in this instance
+      // because at this point Electron will be showing images following the path provided
+      // with the `file:///` protocol -- seems to work
+      const normalizedPath: string = path.normalize(anyOsPath);
+      const forwardSlashes: string = normalizedPath.replace(/\\/g, '/');
```

Comment:
This should already be done everywhere else as of c37341c? Are you still having issues?

Reply:
Before this change, the `globals.selectedOutputFolder` was getting garbled before it got rendered through the template.

This should fix it once and for all 😅